Spigot-API-Patches/0005-Adventure.patch

```diff
@@ -2869,10 +2869,19 @@ index 4ff149fd98895dd8ba45939a37c223b1f8d7281f..3dba4c361993e143e511b4f108ac0b44
 +     */
 +    @NotNull
 +    net.kyori.adventure.text.event.HoverEvent<net.kyori.adventure.text.event.HoverEvent.ShowItem> asHoverEvent(final @NotNull ItemStack item, final @NotNull java.util.function.UnaryOperator<net.kyori.adventure.text.event.HoverEvent.ShowItem> op);
++
++    /**
++     * Get the formatted display name of the {@link ItemStack}.
++     *
++     * @param itemStack the {@link ItemStack}
++     * @return display name of the {@link ItemStack}
++     */
++    @NotNull
++    net.kyori.adventure.text.Component displayName(@NotNull ItemStack itemStack);
```

Comment:
Always found it peculiar that we use JB annotations when Adventure uses Checker framework. Don't gotta change it, just something we should probably consider fixing later on

Reply:
Spigot used JBs when they started annotating stuff, we where still using javax and where discussing on where we wanted to go given that that was gonna break

Reply:
The `AnnotationTest` has been updated to allow checker annotations for return types, but not for parameter types. I looked into updating the test, but it seemed non-trivial. I just went with the JetBrains annotations here so we wouldn't have two types of annotations on the same method.